### ISSUE TITLE
Change resolution of video stream

### ISSUE THREAD

**Issue by grdsvt** [AIZOOTech/FaceMaskDetection#36]
Hi,

**Comment by AIZOOTech** [comment]
I did not do multi scale training. You can change the input to 640, remeber to change the 
```
feature_map_sizes = [[33, 33], [17, 17], [9, 9], [5, 5], [3, 3]]
```
to the corresponding size when input is 640x480. However， The result may not performs well.
The best way is train the model with 640x480 input size.

**Comment by grdsvt** [comment]
Hi, sorry but I don't understand......

**Comment by grdsvt** [comment]
Hi,
changet the resolution to 640x480 and it is working fine.
Do you suggest to re-train the model with this resolution?

**Comment by AIZOOTech** [comment]
Yes, the best solution is to train the model with 640x480 input size.

**Comment by grdsvt** [comment]
I have problem with training......have you this model?

**Comment by AIZOOTech** [comment]
I can train it, however I am busy those days. Are you  using this for commercial application? If so, we can train the new model for you if you can afford some payment.

**Comment by grdsvt** [comment]
no sir, I am studying deep learning to complete my course and I see that your work is very amazing.
Thanks

**Comment by AIZOOTech** [comment]
Ok, this model is a simple SSD framework. Training process is quite easy. You can find many open source SSD implementation on Github. Hope everything is fine.

**Comment by grdsvt** [comment]
Thanks
